reports/Krieger_ddwg_2021.md

```diff
@@ -15,7 +15,7 @@ servers are expected to move to a more permanent home in the future, but the dat
 1. File issues are being tracked down as part of regular maintenance.
 1. Some time is being spent in trying to understand changes in data distribution that have not been communicated to the WG. 
 Explicit, direct and clear communication will be appreciated regarding changes.
-1. A future TODO is to prioritize setting Paul Breen up with an account so that BAS can receive vks, fhe, fhw, gbr and kap data.
+1. A future TODO is to prioritize setting Paul Breen up with an account so that BAS can receive bks, fhe, fhw, gbr and kap data. Initial setup of BAS account was done during SD Workshop.
 1. FITACF 3.0 files no longer shared via Globus
```

Comment:
Did we ever distribute fitacf3 files to USask and globus?  I'm very confused by this comment.

Reply:
Yes we (usask) did briefly due to some requests, but then we stopped due to the SuperDARN community not having consensus on FITACF 3 yet. We have FITACF 2.5 files being shared right now instead.

Reply:
I know you've got this merged in, but thought I would continue here.  Should the update for sharing fitacf 3.0 files then be under updates for USask and not under updates for VT?

Reply:
Thanks for catching that, updated now!